.github/workflows/coverity.yml

```diff
@@ -47,31 +47,74 @@ permissions:
   contents: read
 
 jobs:
-  scan:
-    runs-on: ubuntu-latest
+  coverity-scan:
+    runs-on: ubuntu-22.04
     if: ${{ github.repository_owner == 'apache' }}
 
-    env:
-      DEBIAN_FRONTEND: noninteractive
+    container:
+      image: apache/incubator-cloudberry:cbdb-build-rocky9-latest
+      options: >-
+        --user root
+        -h cdw
 
     steps:
-    - uses: actions/checkout@v4
-    - name: Install apt dependencies
+    - name: Checkout Apache Cloudberry
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 1
+       
+    - name: Environment Initialization
       run: |
-        sudo apt-get update
-        sudo apt-get install -y bison cmake curl flex git-core gcc g++ inetutils-ping krb5-kdc krb5-admin-server libapr1-dev libbz2-dev libcurl4-gnutls-dev libevent-dev libkrb5-dev libpam-dev libperl-dev libreadline-dev libssl-dev libxerces-c-dev libxml2-dev libyaml-dev libzstd-dev locales net-tools ninja-build openssh-client openssh-server openssl pkg-config python3-dev python3-pip python3-psutil python3-pygresql python3-yaml zlib1g-dev
-        pip3 install conan
+        if ! su - gpadmin -c "/tmp/init_system.sh"; then

```

Comment:
where we create the `/tmp/init_system.sh` file?

Reply:
IMO, it's built in the Docker image `apache/incubator-cloudberry:cbdb-build-rocky9-latest`. I verified and it is in the image.